packages/logs/lib/models/helpers.ts

```diff
@@ -140,3 +141,15 @@ export const operationTypeToMessage: Record<ConcatOperationList, string> = {
     'webhook:incoming': 'Received a webhook',
     'webhook:forward': 'Forwarding Webhook'
 };
+
+/**
+ * Send buffered logs to elasticsearch
+ * Ultimately it would be better to have LogContextBuffer (or an option in LogContext)
+ */
+export async function flushLogsBuffer(logs: MessageRowInsert[], logCtx: LogContext) {
```

Comment:
can we not have this function on `LogContext`, for instance `logCtx.flush(logs)` and also have a single call to ES?

Reply:
yes, I think it needs more work as stated in the comment above the function. It's non trivial change so I'll do that later